### ISSUE TITLE
👌 IMPROVE: Information when selecting a work chain

### ISSUE THREAD

**Issue by mbercx** [aiidalab/aiidalab-qe#68]
When selecting a work chain in the "Status & Results" step, the output formation is just the `__repr__` of the work chain:

![Screenshot 2021-06-07 at 15 44 06](https://user-images.githubusercontent.com/25607960/121027407-3b1aa580-c7a7-11eb-815b-a3814066e691.png)

I'd just show the output of `verdi process status` here?

**Comment by mbercx** [comment]
Additional idea from @giovannipizzi: if a calculation fails, it would be good to show `verdi process report` when selecting it here.

**Comment by unkcpz** [comment]
I think we are talking about the default information since for instance `QeAppWorkChain`, has a node view defined for it explicitly. I am up for `verdi process status` for process and `verdi process report` for the failed one.

**Comment by unkcpz** [comment]
I am trying to address this by using the default NodeViewer from aiidalab-widgets-base, see the draft PR https://github.com/aiidalab/aiidalab-widgets-base/pull/430

In the QeApp, the `QeAppWorkChain` has its own specific viewer, and it will now override the viewer mappings for all `WorkChain` class. In the https://github.com/aiidalab/aiidalab-widgets-base/pull/430, it will fallback to the `NodeViewer` (Or probably better more levels, such as Process Viewer first and then NodeViewer). 
If it is a process, the widgets to show the information of process as applied in https://github.com/aiidalab/aiidalab-widgets-base/blob/master/notebooks/process.ipynb all can be used as default which shows exactly as the `verdi process status` and `verdi process show <pk>`.

**Comment by unkcpz** [comment]
I failed to get https://github.com/aiidalab/aiidalab-widgets-base/pull/430 done and have it merged and used here to solve the problem. Since it won't cause critical issue for the app, I remove it from the milestone of this release.

**Comment by danielhollas** [comment]
@unkcpz  I believe the original issue is solved anyway. We now display the logs for workchains, workfunctions, calcfunctions and calcjobs. I personaly do not think https://github.com/aiidalab/aiidalab-widgets-base/pull/430 was a good solution to the problem.

**Comment by unkcpz** [comment]
> We now display the logs for workchains, workfunctions, calcfunctions and calcjobs.

Emm.... Do you mean in your app? How you do it? In the QeApp, click the workchain other than the main workchain shows nothing after. 

> @unkcpz I believe the original issue is solved anyway. 

If you mean the original issue that break the app, yes we solve it by skip when exception raised, but I think this issue to me was ask to use `ProcessReport` widget from `aiidalab-widgets-base` as fallback viewer.

**Comment by danielhollas** [comment]
> Emm.... Do you mean in your app? How you do it? In the QeApp, click the workchain other than the main workchain shows nothing after.

No, it is in AWB. I now remember that QeApp had some code that overwrittes that AWB viewers, because you have a special viewer for the root node. But I think the proper solution is to decouple the main output from the NodeViewer entirely, as I do in my App. That way things should work as expected.
